docs/static/releasenotes.asciidoc

```diff
@@ -3,12 +3,17 @@
 
 This section summarizes the changes in the following releases:
 
+* <<logstash-5-6-5,Logstash 5.6.5>>
 * <<logstash-5-6-4,Logstash 5.6.4>>
 * <<logstash-5-6-3,Logstash 5.6.3>>
 * <<logstash-5-6-2,Logstash 5.6.2>>
 * <<logstash-5-6-1,Logstash 5.6.1>>
 * <<logstash-5-6-0,Logstash 5.6.0>>
 
+[[logstash-5-6-5]]
+=== Logstash 5.6.5 Release Notes
+
```

Comment:
any thoughts of having a note here about why this release is happening?

Reply:
This is a placeholder for 5.6.5 release if and when it happens

Reply:
@jsvd @andrewvc On reflection, I'm going to remove the reference to 5.6.5 here, and wait until if and when a release comes